tests/Geometry/PointTest.php

```diff
@@ -35,23 +37,21 @@ public function testConstructorShouldRequireAtLeastTwoElementsInPosition(): void
         new Point([1]);
     }
 
-    /**
-     * @dataProvider providePositionsWithInvalidTypes
-     */
-    public function testConstructorShouldRequireIntegerOrFloatElementsInPosition(): void
+    #[DataProvider('providePositionsWithInvalidTypes')]
+    public function testConstructorShouldRequireIntegerOrFloatElementsInPosition(array $position): void
     {
         $this->expectException(InvalidArgumentException::class);
         $this->expectExceptionMessage('Position elements must be integers or floats');
 
-        new Point(func_get_args());
+        new Point($position);
```

Comment:
Should this use array unpacking? If so, perhaps the argument name should also be renamed to something like `$pointArgs`.